src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -19,6 +19,17 @@
 
 REPO_CENTRAL = b'https://hg.mozilla.org/mozilla-central'
 REPO_REVIEW = b'https://reviewboard-hg.mozilla.org/gecko'
+MAX_COMMENTS = 30
+MOZREVIEW_COMMENT_SUCCESS = '''
+Static analysis didn't find any C/C++ defects in this patch. Hooray!
+'''
+MOZREVIEW_COMMENT_FAILURE = '''
+Static analysis found {} C/C++ defect{} in this patch{}.
```

Comment:
Thanks! I couldn't hold back one more nit: `C/C++` between the number and the word `defect` is confusing, please move it to the beginning of the sentence, e.g.

```
C/C++ static analysis found {} defect{} in this patch{}.
```